GSA-TTS/FAC
# Bug: PDF report upload does not re-populate page numbers

# Bug: PDF report upload does not re-populate page numbers

Issue: https://github.com/GSA-TTS/FAC/issues/1905

## Changes:
1. The view now searches for an applicable SingleAuditReportFile and pulls it down
    * Existing page numbers are added to the form context
    * `already_submitted` is set if the SAR exists
2. The HTML template mentions that a submission has already been made at the bottom. References to file upload will instead say "re-upload"
3. Cypress does not break.

## How to test:
Swap to this branch and run normally. Navigate to an audit report upload page. Complete the form, come back to it, and verify that the information persisted. 

## Screenshots:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/6f31e539-f9a2-4036-b205-da70d8ce2fdf)
![image](https://github.com/GSA-TTS/FAC/assets/91098850/b4cc95e3-a1e4-4153-9e7c-6884a98cf906)

## PR checklist: submitters


## PR checklist: reviewers

        
The larger the PR, the stricter we should be about these points.
